modules/aerodyn/src/UnsteadyAero_Driver.f90

```diff
@@ -202,7 +202,8 @@ program UnsteadyAero_Driver
    !u(3) = time at n=-1 (t= -2dt) if NumInp > 2
 
    DO iu = 1, NumInp-1 !u(NumInp) is overwritten in time-sim loop, so no need to init here 
-      call setUAinputs(2-iu,  u(iu), uTimes(iu), dt, dvrInitInp, timeArr, AOAarr, Uarr, OmegaArr)
+     call setUAinputs(2-iu,  u(iu), uTimes(iu), dt, dvrInitInp, timeArr, AOAarr, Uarr, OmegaArr, errStat, errMsg) 
+      call checkError()
```

Comment:
The indentation seems weird here, make sure you are not using tabs.

Reply:
I'll adopt using 3 spaces (I normally use 2 or 4).